src/main/java/uk/gov/justice/hmpps/casenotes/config/RestTemplateConfiguration.java

```diff
@@ -29,6 +29,16 @@ public RestTemplate oauthApiRestTemplate(final RestTemplateBuilder restTemplateB
         return getRestTemplate(restTemplateBuilder, properties.getOauthApiBaseUrl());
     }
 
+    @Bean(name = "elite2ApiRestTemplateWithTimeout")
```

Comment:
not a great fan of this naming - would prefer `elite2HealthRestTemplate` or similar since it is only used for health checks